modules/java_api/build.gradle

```diff
@@ -14,7 +14,7 @@ def nativesCPP;
 def openvinoVersion = "2024.2"
 
 def native_resources = []
-def tbb_dir = System.getenv('TBB_DIR')
+def tbb_dir = System.getenv('TBB_DIR') ?: System.getenv('OPV_HOME_DIR') + "/one-tbb-install/lib/cmake/TBB"
```

Comment:
one-tbb-install directory can be changed, it's just example and there cannot be a universal way to find TBB, that why System.getenv('TBB_DIR') is required